src/mrsimulator/load_vendor_files.py

```diff
@@ -0,0 +1,114 @@
+# -*- coding: utf-8 -*-
+import csdmpy as cp
+import nmrglue as ng
+
+
+def load_bruker_1d(filename):
+    dic, data = ng.bruker.read(filename)
+    udic = ng.bruker.guess_udic(dic, data)
+
+    dims = list()
+    for key, value in list(udic.items()):
+        if type(key) == int:
+            obj = cp.Dimension(
+                type="linear",
+                count=value["size"],
+                increment=(str(1 / value["sw"]) + " s"),
+                coordinates_offset=(str(1 / value["car"]) + " s"),
+                origin_offset=(str(1e-6 / value["obs"]) + " s"),
```

Comment:
replace these lines with

reciprocal={"coordinates_offset": f'{value["car"]} Hz', "origin_offset": f'{value["obs"]} MHz'} 

The parameter belong in the reciprocal  object.